test/conftest.py

```diff
@@ -38,15 +48,18 @@ def insert_data_from_csv(in_csv_file, rec_count=-1):
             "./input/bboxes-constraints.adb.csv": BBoxDataCSV,
             "./input/gs_images.adb.csv": ImageDataCSV
         }
-
-        data = file_data_pair[in_csv_file](in_csv_file)
+        use_dask = False
+        if hasattr(request, "param"):
+            use_dask = request.param
+        data = file_data_pair[in_csv_file](in_csv_file, use_dask=use_dask)
         if rec_count != -1:
             data = data[:rec_count]
 
         loader = ParallelLoader(db)
         loader.ingest(data, batchsize=99,
-                      numthreads=31,
```

Comment:
I would revert this change, so that we tests with stranger number of threads. This used to catch certain weird sitautions